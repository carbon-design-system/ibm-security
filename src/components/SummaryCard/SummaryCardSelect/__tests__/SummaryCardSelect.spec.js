/**
 * @file Summary card select tests.
 * @copyright IBM Security 2019
 */

import { render } from '@testing-library/react';
import React from 'react';

import { SummaryCardSelect } from '../../../..';

import props from '../_mocks_';

describe('SummaryCardSelect', () => {
  it('renders', () => {
    expect(
      render(<SummaryCardSelect className="className" {...props} />).container
        .firstChild
    ).toMatchSnapshot();
  });
});
