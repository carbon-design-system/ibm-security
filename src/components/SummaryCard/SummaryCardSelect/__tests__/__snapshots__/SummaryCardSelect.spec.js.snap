// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SummaryCardSelect renders 1`] = `
<div
  class="security--summary-card__select"
>
  <div
    class="bx--form-item bx--checkbox-wrapper"
  >
    <input
      class="bx--checkbox"
      id="summaryCardSelect"
      type="checkbox"
    />
    <label
      class="bx--checkbox-label security--summary-card__select__checkbox className"
      for="summaryCardSelect"
    >
      <span
        class="bx--checkbox-label-text"
      >
        Select
      </span>
    </label>
  </div>
</div>
`;
